charter.html

```diff
@@ -193,9 +193,9 @@ <h2 id="liaisons">
     <ul>
       <li>
         <a href=
-        "https://github.com/whatwg/sg/blob/master/Workstreams.md#html">HTML</a>
+        "https://whatwg.org/workstreams#html">HTML</a>
```

Comment:
The new workstreams page doesn't include IDs for the different streams, so these anchors don't resolve.

Reply:
We fixed this in: https://github.com/whatwg/whatwg.org/commit/839ca039f8e5b8c7ab9c2e04dafb18282f45b4d6

I'll check what happened on a new build that removed it. Please keep this PR open.

@yoavweiss

Reply:
![image](https://user-images.githubusercontent.com/11020807/205284718-6f8d4998-696d-4464-903b-392583a76bf5.png)

It's there buddy!

Reply:
Awesome, thanks!! :)